src/common/service-dependency.ts

```diff
@@ -0,0 +1,212 @@
+import execa = require('execa');
```

Comment:
See `src/commands/deploy.ts` for details on how to use new dependency-manager resource for graphing. If you need details like `api_definitions` or `language`, please add them to the node data structures.